libpldmresponder/fru.cpp

```diff
@@ -544,6 +544,13 @@ void FruImpl::buildIndividualFRU(const std::string& fruInterface,
     pldm_entity parentEntity{};
     static uint32_t last_bmc_record_handle = 0;
     uint32_t newRecordHdl{};
+
+    if(fruInterface == "xyz.openbmc_project.Inventory.Item.PCIeDevice")
+    {
+        info("Removing old PDRs {PATH}", "PATH", fruObjectPath);
+        removeIndividualFRU(fruObjectPath);
```

Comment:
if we remove this will we not lose the LED state values corresponding the card and all? (not sure though)